app/controllers/steps/details/check_answers_controller.rb

```diff
@@ -0,0 +1,12 @@
+module Steps::Details
+  class CheckAnswersController < Steps::DetailsStepController
+    def show
+      raise 'No tribunal case in session' unless current_tribunal_case
+
+      @fee_details_answers = FeeDetailsAnswersPresenter.new(current_tribunal_case)
+      @appeal_timeliness_answers = AppealTimelinessAnswersPresenter.new(current_tribunal_case)
+      @appellant_details = AppellantDetailsPresenter.new(current_tribunal_case)
+      @documents_submitted = DocumentsSubmittedPresenter.new(current_tribunal_case)
```

Comment:
See https://dsdmoj.atlassian.net/wiki/display/TT/Glossary where we have a list of the terms